scripts/cm_test_all_sandia

```diff
@@ -731,7 +731,6 @@ elif [ "$MACHINE" = "vega90a_caraway" ]; then
     ARCH_FLAG="--arch=VEGA90A"
   fi
 elif [ "$MACHINE" = "blake" ]; then
-  MODULE_ENVIRONMENT="source /etc/profile.d/modules.sh"
```

Comment:
All the modules on blake will need to be updated for blake to be usable in the script, these are all pre-DST hardware (and software) upgrades. I can put in a separate PR for that

Reply:
@ndellingwood and I discussed verbally - the modules are currently handled in the Jenkins job definition as a workaround while solo is down